common/utils/src/main/resources/error/error-classes.json

```diff
@@ -674,6 +674,12 @@
     ],
     "sqlState" : "23505"
   },
+  "DUPLICATE_ROUTINE_PARAMETER_ASSIGNMENT" : {
+    "message" : [
+      "Call to function <functionName> is invalid because it includes multiple argument assignments to the same parameter name <parameterName>."
+    ],
+    "sqlState" : "4274K"
```

Comment:
Let's clarify the sqlState with @srielau . The sqlState used in the error classes should be recorded in https://github.com/apache/spark/tree/master/common/utils/src/main/resources/error. According to https://www.ibm.com/docs/en/i/7.4?topic=codes-listing-sqlstate-values the `4274K` seems quite valid, should we document it in the doc https://github.com/apache/spark/tree/master/common/utils/src/main/resources/error?

Reply:
Yep, consulted with him, and I have added the 4274K state documentation to the table.